doc/api/cli.md

```diff
@@ -178,6 +178,14 @@ added: v8.0.0
 Enable loading native modules compiled with the ABI-stable Node.js API (N-API)
 (experimental).
 
+## `--abort-on-uncaught-exception`
+<!-- YAML
+added: v0.10
+-->
+
+Aborting instead of exiting causes a core file to be generated for post-mortem
+analysis using a debugger (such as `lldb`, `gdb`, and `mdb`).
```

Comment:
Yes, should I give some advice here on ulimit, or just say that it may generate a core file?